### ISSUE TITLE
Suggestion: add acsuite and lvsfunc plugins

### ISSUE THREAD

**Issue by OrangeChannel** [4re/vapoursynth-portage#89]
<https://github.com/OrangeChannel/acsuite>
<https://github.com/Irrational-Encoding-Wizardry/lvsfunc>

For parity with the aur:
<https://forum.doom9.org/showthread.php?t=168521>
<https://bitbucket.org/James1201/vapoursynth-cmedian/src/default/>
<https://github.com/tormaid/colorfade>
<https://github.com/chikuzen/convo2d>
<https://github.com/WolframRhodium/VapourSynth-dpid>
<https://github.com/sekrit-twc/EdgeFixer>
<https://forum.doom9.org/showthread.php?t=171741>
<https://forum.doom9.org/showthread.php?t=172652>
<https://gist.github.com/SX91/bcd427ec0fa8fdf7c45433917521bac4>
<https://github.com/IFeelBloated/vapoursynth-focus/releases>
<https://github.com/xekon/GradCurve>
<https://gist.github.com/4re/34ccbb95732c1bef47c3d2975ac62395>
<https://github.com/sekrit-twc/imagine>
<https://gist.github.com/4re/b5399b1801072458fc80>
<https://github.com/MonoS/MonoS-VS-Func>
<https://github.com/MonoS/VS-MaskDetail>
<https://github.com/HomeOfAviSynthPlusEvolution/neo_f3kdb>
<https://bitbucket.org/James1201/vapoursynth-noisegen/src/default/>
<https://gist.github.com/4re/2545a281e3f17ba6ef82>
<https://github.com/4re/resamplehq>
<https://github.com/IFeelBloated/RGSF>
<https://gist.github.com/4re/3acdb8d90abf78643d4e>
<https://gist.github.com/4re/8fee7aa32ba9ac566ed0>
<https://gist.github.com/4re/bba3f65469acfe0ec08a>

**Comment by 4re** [comment]
I will look into it, but this will take me some time. I'm short on it until December.

**Comment by OrangeChannel** [comment]
Awesome, thanks! 👍

**Comment by OrangeChannel** [comment]
lvsfunc has been re-structured over the past few days and is no longer just a file in the repo, it's a "proper" looking Python package now, so not sure how/if that affects the portage build script but figured I might as well mention it here (since the AUR pkgbuild also needs to be redone).

Also, now requires py3.8 and VS 49+ since it uses spline64.

<https://github.com/Irrational-Encoding-Wizardry/lvsfunc>

**Comment by 4re** [comment]
Yeah, first day I did the changes on the live ebuild but decided not to make any release until one week of inactivity on that repo or so, because it has no proper releases/tags and to make sure nothing is too broken.
It looks like it's working now but thanks anyway for the heads-up.
